pmm-app/src/pmm-dbaas/components/DBCluster/AddDBClusterModal/DBClusterAdvancedOptions/DBClusterAdvancedOptions.tsx

```diff
@@ -34,7 +34,13 @@ export const DBClusterAdvancedOptions: FC<FormRenderProps> = ({
   const { required, min } = validators;
   const { change } = form;
   const nodesValidators = [required, min(MIN_NODES)];
-  const resourcesValidators = [required, min(MIN_RESOURCES)];
+  const diskValidators = [required, min(MIN_RESOURCES)];
+  const resourcesValidators = [
+    required,
+    min(0.1),
+    (value) => ((value * 10) % 1 ? 'Value must be increments of 0.1.' : undefined),
```

Comment:
nit: create a validator function for such cases. It's very useful.

Reply:
Thanks, already moved it to another PR

Reply:
removed, was added by mistake